lib/babel-config.js

```diff
@@ -22,7 +26,12 @@ function validate(conf) {
 		return {testOptions: {}};
 	}
 
-	if (!conf || typeof conf !== 'object' || !conf.testOptions || typeof conf.testOptions !== 'object' || Array.isArray(conf.testOptions) || Object.keys(conf).length > 1) {
+	let invalid = false;
+
+	invalid = !conf || typeof conf !== 'object' || !conf.testOptions || typeof conf.testOptions !== 'object' || Array.isArray(conf.testOptions);
+	invalid = !validateKeys(conf) || (conf.extensions && !Array.isArray(conf.extensions));
```

Comment:
This second line needs to start with `invalid = invalid || …`.

It'll be clearer to put the condition in an `if` clause and then explicitly assign `true`.

Reply:
I placed these into an if statement like you recommended